packages/nextjs/utils/scaffold-eth/useNetworkColor.ts

```diff
@@ -1,6 +1,10 @@
 import { getTargetNetwork } from "./networks";
 import { useDarkMode } from "usehooks-ts";
 
+/**
+ * @dev Gives you the color of the current configured/target network
+ * @return {string | [string, string]} network color/colors
```

Comment:
I think this never returns an array, right?

Reply:
oops, you're right, thanks :)